tst/EnergyPlus/unit/DOASEffectOnZoneSizing.unit.cc

```diff
@@ -215,6 +215,7 @@ TEST_F(EnergyPlusFixture, DOASEffectOnZoneSizing_SizeZoneEquipment)
     state->dataSize->CalcZoneSizing(state->dataSize->CurOverallSimDay, 2).DOASHighSetpoint = 14.4;
     state->dataSize->CalcZoneSizing(state->dataSize->CurOverallSimDay, 2).DOASLowSetpoint = 12.2;
     state->dataEnvrn->StdBaroPress = 101325.;
+    state->dataEnvrn->StdRhoAir = 1.0;
```

Comment:
Set air density to fix a failed unit test. The air density was set to 1.0 avoid introducing additional diffs.

Reply:
Got it.  This is a clever change to get the effect in there without breaking the unit test :)